localstack-core/localstack/openapi.yaml

```
@@ -668,6 +702,8 @@ paths:
   /_localstack/health:
     get:
       description: Get available LocalStack features and AWS services
+      operationId: get_ls_features_and_services
```

```suggestion
      operationId: get_features_and_services
```

`ls` is already implied via the tag